codegen/service/service.go

```diff
@@ -205,6 +206,15 @@ type Service interface {
 {{- end }}
 }
 
+{{- if .Schemes }}
+// authenticated is the internal interface for security functions
+type authenticated interface {
```

Comment:
`auther` might be a bit more idiomatic?
```suggestion
type auther interface {
```